api/v1beta1/types.go

```diff
@@ -893,6 +906,17 @@ func (s SubnetSpec) IsIPv6Enabled() bool {
 	return false
 }
 
+// GetSecurityRuleByDestination returns security group rule, which matches provided destination ports.
+func (s SubnetSpec) GetSecurityRuleByDestination(ports string) *SecurityRule {
```

Comment:
`port` seems to be holding a single port here.
```suggestion
func (s SubnetSpec) GetSecurityRuleByDestination(port string) *SecurityRule {
```

Reply:
I doublechecked it, maybe my initial assumption was wrong. It can be a number or an `*` there, so changed it to port as suggested.